Shared/Samples/Display content of utility network container/DisplayContentOfUtilityNetworkContainerView.Model.swift

```diff
@@ -0,0 +1,278 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import Combine
+import UIKit
+
+extension DisplayContentOfUtilityNetworkContainerView {
+    /// The model used to manage the state of the trace view.
+    @MainActor
+    class Model: ObservableObject {
+        // MARK: Properties
+        
+        /// The status message shown to the user.
+        @Published var statusMessage = ""
+        
+        /// The Naperville Electric Containers web map.
+        let map = Map(item: .napervilleElectricalNetwork())
+        
+        /// The graphics overlay to display utility network graphics.
+        let graphicsOverlay = GraphicsOverlay()
+        
+        /// A line symbol to show the bounding extent.
+        private let boundingBoxSymbol = SimpleLineSymbol(style: .dash, color: .yellow, width: 3)
+        
+        /// A line symbol to show the attachment association.
+        private let attachmentSymbol = SimpleLineSymbol(style: .dot, color: .green, width: 3)
+        
+        /// A line symbol to show the connectivity association.
+        private let connectivitySymbol = SimpleLineSymbol(style: .dot, color: .red, width: 3)
+        
+        /// The feature layers that allow us to fetch the legend symbols of
+        /// different elements in the network.
+        private let featureLayers: [FeatureLayer] = {
+            let layerIDs = [1, 5]
+            return layerIDs.map { layerID in
+                let url: URL = .featureService.appendingPathComponent("\(layerID)")
+                let table = ServiceFeatureTable(url: url)
+                return FeatureLayer(featureTable: table)
+            }
+        }()
+        
+        /// The utility network for this sample.
+        private let network: UtilityNetwork
+        
+        /// The legends for elements in the utility network.
+        @Published private(set) var legendItems: [LegendItem] = []
```

Comment:
Instead of making this non-private, you could keep it private and add a computed property:

```swift
var hasLegendItems: Bool { !legendItems.isEmpty }
```

That way, the view only has the information it needs, since it never needs the legend items themselves.